TEveEventDisplay/src/TEveMu2eTracker.cc

```diff
@@ -19,97 +16,79 @@ namespace mu2e{
       Double_t rmin{pointmmTocm(envelope.innerRadius())};
       Double_t rmax{pointmmTocm(envelope.outerRadius())};
         
-      //Tracker Planes in XZ
+      //Tracker Stations in XZ
       double p = 0.0;
-      for(int i =0;i<20;i++)
-        {
-        Double_t panelpos[3];
+      //unsigned int nStations = trkr->nPlanes()/2;
+      unsigned int dp = 0;
+      Double_t stationpos[3];
+      for(unsigned int i =0; i < 20 ;i++){
```

Comment:
What does 20 mean?  it should be a symbolic constant

Reply:
yes, dont worry, as I said in my email....Namitha is working on these fixes, we should use nStations which is in fact 18 and she is aware of this...she will put in her PR in the next few days